pkger/parser_test.go

```diff
@@ -1833,7 +2078,7 @@ spec:
 			})
 		})
 
-		t.Run("single xy chart", func(t *testing.T) {
+		t.Run("pkg with single dashboard xy chart", func(t *testing.T) {
```

Comment:
please change this back to the what it was before, this creates stutter in the test output. The test this is nested in provides the context of `pkg with single dashboard and single chart`.

Reply:
that's my bad. I must've accepted the wrong diff during my rebase. will fix